onecgiar-pr-client/src/app/pages/results/pages/result-detail/pages/rd-theory-of-change/components/shared/toc-initiative-out/target-indicator/target-indicator.component.scss

```diff
@@ -148,44 +102,98 @@ input:disabled {
         margin-right: 5px;
       }
 
-      .info-messurent-item {
+      .target_header_info-item {
         word-wrap: break-word;
       }
     }
+  }
+
+  .indicator_content_information {
+    padding: 1rem 1.2rem;
 
-    .section-two {
+    .indicator_content_information_item {
       display: flex;
-      flex-direction: row;
-      padding-left: 10px;
-      margin-top: -25px;
+      justify-content: space-between;
+      align-items: flex-start;
+      gap: 1rem;
 
-      .overall-target {
-        display: flex;
-        flex-direction: row;
+      @media screen and (max-width: 1140px) {
+        flex-wrap: wrap-reverse;
       }
     }
   }
+}
 
-  td,
-  th {
-    border: 1px solid #dddddd;
-    text-align: center;
-    padding: 8px;
-  }
+.tiny_line {
+  background-color: #d9d9d9;
+  width: 100%;
+  height: 2px;
+  margin-top: 20px;
+  margin-bottom: 0px;
+  border-radius: 5px;
+}
 
-  .input-not-calculable {
-    width: 100%;
-    height: 70px;
-    border-radius: 5px;
-    border: 1px solid #9999999a;
-    margin-bottom: 10px;
-    margin-top: 10px;
-    font-size: 14px;
-    color: #808080;
-    font-weight: 500;
+.line {
+  background-color: #d9d9d9;
+  width: 100%;
+  height: 5px;
+  margin-top: 20px;
+  margin-bottom: 0px;
+  border-radius: 5px;
+}
 
-    &:focus {
-      outline: none;
-    }
-  }
+.bigger-line {
+  background-color: #d9d9d9;
+  width: 100%;
+  height: 10px;
+  border-radius: 0 0 10px 10px;
+  display: flex;
+  align-items: center;
 }
+
+// Borrar
```

Comment:
@xKeCo could you check this commented code block

Reply:
Done ✅